src/schemas/json/ruff.json

```diff
@@ -818,7 +818,7 @@
       "type": "object",
       "properties": {
         "convention": {
-          "description": "Whether to use Google-style or Numpy-style conventions when detecting docstring sections. By default, conventions will be inferred from the available sections.",
+          "description": "Whether to use Google-style, Numpy-style or PEP257-style conventions when detecting docstring sections. By default, conventions will be inferred from the available sections.",
```

Comment:
Mandatory fix: [broken description](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices): `\nDOCUMENTATION_URL` expected at the end of description.